sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -719,6 +716,8 @@ public virtual async Task StopProcessingAsync(CancellationToken cancellationToke
         internal async Task UpdateCheckpointAsync(EventData eventData,
                                                   PartitionContext context)
         {
+            // TODO: Should this method take a cancellation token from the ProcessEventArgs?
```

Comment:
You know.... I hadn't thought of that.   It probably should.   I'll note that for Monday, if you don't get to it first.